tests/test_twisted.py

```diff
@@ -184,6 +184,19 @@ def test_catchesKeyCertificateMismatch(self, tmpdir):
 
         assert str(excinfo.value).startswith("No certificate matching ")
 
+    def test_catchesMultipleDHParams(self, tmpdir):
+        pemFile = tmpdir.join('multiple_params.pem')
+        pemFile.write(KEY_PEM + CERT_PEMS[0] + DH_PEM + DH_PEM)
+
+        with pytest.raises(ValueError) as excinfo:
+            certificateOptionsFromFiles(
+                str(pemFile)
+            )
+
+        assert str(excinfo.value) == (
```

Comment:
I changed this but I wasn't entirely sure how to format the whole multiline string + operator thing, let me know if you don't like what I did.